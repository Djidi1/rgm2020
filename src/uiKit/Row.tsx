import styled from 'styled-components';

interface RowProps {
  margin?: string
}

const Row = styled.div<RowProps>`
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  ${({ margin }) => (margin ? `margin: ${margin};` : '')}
`;

export default Row;
