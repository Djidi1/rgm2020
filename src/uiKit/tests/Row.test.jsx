import React from 'react';
import renderer from 'react-test-renderer';
import Row from '../Row';

describe('UI-Kit Row', () => {
  beforeEach(() => {
    jest.resetModules();
  });

  it('is expected to render successfully', () => {
    const wrapper = renderer
      .create(<Row />)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
