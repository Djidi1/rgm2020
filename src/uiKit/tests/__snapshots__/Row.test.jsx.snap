// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI-Kit Row is expected to render successfully 1`] = `
<div
  className="Row-sc-1g9of81-0 buElqX"
/>
`;
